src/main/java/ua/com/shop/controller/OrderController.java

```diff
@@ -0,0 +1,63 @@
+package ua.com.shop.controller;
+
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import ua.com.shop.dto.CreateOrderDto;
+import ua.com.shop.model.Order;
+import ua.com.shop.model.OrderInfo;
+import ua.com.shop.model.Product;
+import ua.com.shop.service.OrderService;
+import ua.com.shop.service.ProductService;
+
+import java.util.List;
+
+@RequestMapping("/orders")
+public class OrderController {
+
+    private ProductService productService;
+    private OrderService orderService;
+
+    public OrderController(ProductService productService, OrderService orderService) {
```

Comment:
(suggestion:Hybris coding style) please use get/set methods to inject dependencies